components/Repo/Add.js

```diff
@@ -10,17 +10,42 @@ import {
   Field,
   Button,
   Dropdown,
-  mediaQueries
+  Autocomplete,
+  mediaQueries,
+  colors,
+  Loader
 } from '@project-r/styleguide'
 
 import { GITHUB_ORG, TEMPLATES, REPO_PREFIX } from '../../lib/settings'
+import gql from 'graphql-tag'
+import { compose, graphql } from 'react-apollo'
+import { withRouter } from 'next/router'
+import SearchIcon from 'react-icons/lib/md/search'
 
-let templateKeys = Object.keys(schemas)
+const getTemplateRepos = gql`
+  query templateListSearch {
+    reposSearch(isTemplate: true) {
+      totalCount
+      nodes {
+        id
+        latestCommit {
+          id
+          document {
+            id
+            meta {
+              title
```

Comment:
We also need the `meta.template` (aka schema) for the `getSlug` function to add a proper repoPrefix:
```suggestion
              title
              template
```

See related suggestion at auto complete.

Reply:
Since we decided to use the template name as prefix, I assume this change to be obsolete.

Reply:
aaah it's in Add.js... my bad

Reply:
`template` is still missing but used in the auto complete